src/applications/hca/tests/actions.unit.spec.js

```diff
@@ -0,0 +1,235 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import {
+  mockApiRequest,
+  mockFetch,
+  resetFetch,
+} from 'platform/testing/unit/helpers.js';
+
+import {
+  getDismissedHCANotification,
+  setDismissedHCANotification,
+  getEnrollmentStatus,
+  FETCH_ENROLLMENT_STATUS_STARTED,
+  FETCH_ENROLLMENT_STATUS_SUCCEEDED,
+  FETCH_ENROLLMENT_STATUS_FAILED,
+  FETCH_DISMISSED_HCA_NOTIFICATION_STARTED,
+  FETCH_DISMISSED_HCA_NOTIFICATION_SUCCEEDED,
+  FETCH_DISMISSED_HCA_NOTIFICATION_FAILED,
+  SET_DISMISSED_HCA_NOTIFICATION,
+} from '../actions';
+import { beforeEach } from 'mocha';
+
+function setFetchResponse(stub, data) {
+  const response = new Response(null, {
+    headers: { 'content-type': ['application/json'] },
+  });
+  response.ok = true;
+  response.json = () => Promise.resolve(data);
+  stub.resolves(response);
+}
+
+const originalFetch = global.fetch;
+let dispatch;
+
+describe('HCA actions', () => {
+  describe('getEnrollmentStatus', () => {
+    beforeEach(() => {
+      dispatch = sinon.spy();
```

Comment:
Totally not necessary, but I think you can use `dispatch.resetHistory()` here have `dispatch` set to `const dispatch = sinon.spy()` in the parent scope.

Reply:
oh nice, I didn't know about `resetHistory()`